src/runtime/config/configuration-acrn.toml.in

```diff
@@ -117,7 +117,12 @@ block_device_driver = "@DEFBLOCKSTORAGEDRIVER_ACRN@"
 # Warnings will be logged if any error is encountered while scanning for hooks,
 # but it will not abort container execution.
 #guest_hook_path = "/usr/share/oci/hooks"
-
+#
+# Hook timeout is the number of seconds before aborting the hook.
+# If use the OCI Hook, user can set a hook timeout to ensure that the hook has enough time to execute.
+# If set, it must be greater than zero. default value is 10s
```

Comment:
1. The guest hook is a kata-defined hook. It is not an OCI hook.
2. We should allow zero timeout value for those who aren't sure about how long their hooks are expected to execute.

I'd suggest something like this:
```
Guest hook timeout is the number of seconds to wait for a hook execution before aborting it.
It must be a non-negative integer. Zero means no timeout. The default value is 10 seconds.
```

Reply:
@bergwolf done